client/verta/verta/_internal_utils/_utils.py

```diff
@@ -283,16 +283,21 @@ def find(self, where):
         --------
         .. code-block:: python
 
+            runs.find("hyperparameters.hidden_size == 256",
+                       "metrics.accuracy >= .8")
+            # <ExperimentRuns containing 3 runs>
+            # alternatively:
             runs.find(["hyperparameters.hidden_size == 256",
                        "metrics.accuracy >= .8"])
             # <ExperimentRuns containing 3 runs>
 
         """
-        new_list = copy.deepcopy(self)
+        if len(args) == 1 and isinstance(args[0], list):
+            # to keep backward compatibility, in case user pass in a list
+            return self.find(*args[0])
 
```

Comment:
To be extra extra cautious, I think we should also add an `elif` that checks that all `args` are `str`
```
elif not all():  # isinstance(, six.string_types)
```

Reply:
Typecheck added. Also add a check in the test to verify new behavior.